json-sports-team/index.js

```diff
@@ -1,178 +1,257 @@
+const teamName = document.getElementById("team");
+const sport = document.getElementById("sport");
+const year = document.getElementById("year");
+const headCoach = document.getElementById("head-coach");
+const matches = document.getElementById("matches");
+const playerCards = document.getElementById("player-cards");
+const selectElement = document.getElementById("players");
+
+/**
+ * I don't know if we want them to build out this JSON data or just supply it to them in the first step.
+ * If we supply them with the myTeam JSON then we can have the first few steps be about reviewing how to access property values.
+ * Ex. console.log(myTeam.Sport);
+ * We could have them review dot and bracket notation.
+ * This will also be good review for nested objects.
+ * Ex. console.log(myTeam["Head Coach"].Name);
+ */
+
 const myTeam = {
-  "Team": "Argentina",
-  "Sport": "Football",
-  "Year": 1986,
+  Team: "Argentina",
+  Sport: "Football",
+  Year: 1986,
   "Is World Cup Winner": true,
   "Head Coach": {
-    "Name": "Carlos Bilardo",
-    "Matches": 7
+    Name: "Carlos Bilardo",
+    Matches: 7,
   },
-  "Players": [
+  Players: [
     {
-      "Name": "Sergio Almirón",
-      "Position": "FW",
-      "Number": 1,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Sergio Almirón",
+      Position: "FW",
```

Comment:
Having abbreviated positions works, but it might be good to spell these out so people unfamiliar with football can get a sense of what they mean.